kuma/static/js/dashboard.js

```diff
@@ -6,15 +6,102 @@
     var $filterForm = $('#revision-filter');
     var $pageInput = $('#revision-page');
     var currentLocale = $('html').attr('lang');
-    var controlsTemplate = '' +
-    '<div class="action-bar">' +
-        '<ul id="page-buttons">' +
-            '<li><a id="revert" href="$reverturl" class="button">' + gettext('Revert') + '<i aria-hidden="true" class="icon-undo"></i></a></li>' +
-            '<li><a id="view" href="$viewurl" class="button">' + gettext('View Page') + '<i aria-hidden="true" class="icon-circle-arrow-right"></i></a></li>' +
-            '<li class="page-edit"><a id="edit" href="$editurl" class="button">' + gettext('Edit Page') + '<i aria-hidden="true" class="icon-pencil"></i></a></li>' +
-            '<li class="page-history"><a id="history" href="$historyurl" class="button">' + gettext('History') + '<i aria-hidden="true" class="icon-book"></i></a></li>' +
-        '</ul>' +
-    '</div>';
+
```

Comment:
I think this is the JS for the revision dashboard. It looks OK (yep, that's JavaScript), but I wasn't able to view it due to [bug 1419419](https://bugzilla.mozilla.org/show_bug.cgi?id=1419419)

Reply:
I was able to see this luckily and it works as it does currently.